README.md

```diff
@@ -61,7 +61,13 @@
 
 6) Execution Context
 
-   Support to build an OS application for spdm_requester_emu and spdm_responder_emu to trace communication between Requester and Responder.
+   Support to build an OS application for spdm_requester_emu and spdm_responder_emu to trace communication between Requester and Responder in [spdm-emu](https://github.com/DMTF/spdm-emu).
+
+   Support to build an OS application for SPDM validation in [SPDM-Responder-Validator](https://github.com/DMTF/SPDM-Responder-Validator) and TEE-IO validation in [tee-io-validator](https://github.com/intel/tee-io-validator).
+
+   Support to build as part of kernel module in [open-gpu-kernel-modules](https://github.com/NVIDIA/open-gpu-kernel-modules).

```

Comment:
"as part of the NVIDIA Linux kernel module driver"

Reply:
fixed